starter-code/js/index.js

```diff
@@ -1,17 +1,35 @@
+var quantities = document.getElementsByClassName("amount-value");
+var initialPrices = document.getElementsByClassName("unit-price-number");
+var totalPrices = document.getElementsByClassName("final-price-value");
+
+var product = document.getElementsByClassName("product")
+
 function deleteItem(e){
 
 }
 
 function getPriceByProduct(itemNode){
-
+  return itemNode.innerText;
 }
 
-function updatePriceByProduct(productPrice, index){
+function getQuantityByProduct(itemNode) {
+  return itemNode.valueAsNumber;
+}
 
+function multiplyPerQuantityAndUpdate(productPrice, quantity, index){
```

Comment:
We maybe can split this function in two. One for the logic of multiply and another one for the DOM element manipulation.